index.html

```diff
@@ -114,6 +114,18 @@
       </ul>
     </section>
 
+    <section id='intent'>
```

Comment:
I would suggest merging this new text with the "Abstract" section, and renaming the section to "Introduction".